Jenkinsfile

```diff
@@ -242,25 +242,6 @@ timestamps {
                     }
                 }
 
```

Comment:
Removed the integration tests from the Jenkins workflow.  Can I take it to mean that we have these test running elsewhere?

Reply:
Theyre all now in the services repository and will run as part of the services mvn install